src/commands/get_balance.ts

```diff
@@ -2,14 +2,19 @@ import { cliWalletDao } from '..';
 import { CLICommand, ParametersHelper } from '../CLICommand';
 import { AddressParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
 
+const anArInWinston = 1_000_000_000_000;
+const numberOfDigitsAfterComma = 8;
+
 new CLICommand({
 	name: 'get-balance',
 	parameters: [WalletFileParameter, SeedPhraseParameter, AddressParameter],
 	async action(options) {
 		const parameters = new ParametersHelper(options);
 		const address = await parameters.getWalletAddress();
-		const balance = await cliWalletDao.getAddressWinstonBalance(address);
-		console.log(balance);
+		const balanceInWinston = await cliWalletDao.getAddressWinstonBalance(address);
+		const balanceInAR = balanceInWinston / anArInWinston;
```

Comment:
Consider using `arweave.ar.winstonToAr` instead.

Reply:
Here it is: 84da1c34155b75342e49264d08cf55eaf9601b8b